app/controllers/TagManagementApi.scala

```diff
@@ -81,6 +81,19 @@ object TagManagementApi extends Controller with PanDomainAuthActions {
     }.getOrElse(NotFound)
   }
 
+  def createSection() = APIAuthAction { req =>
```

Comment:
Great catch!

On Fri, 22 Jan 2016 at 17:26, Sam Cutler <notifications@github.com> wrote:

> In app/controllers/TagManagementApi.scala
> <https://github.com/guardian/tagmanager/pull/85#discussion_r50564670>:
>
> > @@ -81,6 +81,19 @@ object TagManagementApi extends Controller with PanDomainAuthActions {
> >      }.getOrElse(NotFound)
> >    }
> >
> > +  def createSection() = APIAuthAction { req =>
>
> Is there no permission requirement on creating sections?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/tagmanager/pull/85/files#r50564670>.
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396